plugin/lspconfig.lua

```diff
@@ -28,23 +28,48 @@ local lsp_complete_configured_servers = function(arg)
   end, util.available_servers()))
 end
 
-local lsp_get_active_client_ids = function(arg)
+local lsp_get_active_clients = function(arg)
   local clients = vim.tbl_map(function(client)
-    return ('%d:%s'):format(client.id, client.name)
+    return ('%s'):format(client.name)
   end, util.get_managed_clients())
 
   return completion_sort(vim.tbl_filter(function(s)
     return s:sub(1, #arg) == arg
   end, clients))
 end
 
+---@return vim.lsp.Client[] clients
 local get_clients_from_cmd_args = function(arg)
   local result = {}
+  local managed_clients = util.get_managed_clients()
+  local clients = {}
+  for _, client in pairs(managed_clients) do
+    clients[client.name] = client
+  end
+
+  local err_msg = ''
+  arg = arg:gsub('[%a-_]+', function(name)
```

Comment:
Is this enough to match all valid config names ?

Reply:
nope.. some server file name without underline..

Reply:
But this matches server names without underscore too so that is not a problem. I meant if there are characters that I'm currently not matching that should be included.